### DISCUSSION TITLE
Release 1.5.0

### DISCUSSION THREAD

**Discussion by JMoserCricut** [argoproj/argo-rollouts#2760] [General]
@zachaller When is 1.5.0 going to get a release? according to the [roadmap](https://argoproj.github.io/argo-rollouts/roadmap/) it was supposed to be out April 10:
![image](https://user-images.githubusercontent.com/59970772/236577113-2945ec16-959f-496a-a9c8-8e65a23cffc5.png)

We're anticipating using the feature about the multiple nginx ingresses

**Comment by zachaller** [comment]
I actually have it built and will be releasing it on Monday, sorry for the delay and not keeping the schedule up to date.
